docs/ibc/Overview.md

```diff
@@ -31,28 +31,28 @@ Provider side and the Consumer side. This is done in multiple stages.
 
 Establish a channel (allow list):
 
-1. Admin deploys _Receiver_ on Consumer side.
-2. Admin deploys _External Staker_ on Provider side, registering (connection, port) from _Receiver_ contract from (1)
-3. Admin updates _Receiver_ (1) with the (connection, port) from (2)
-4. Anyone establishes a new channel between _Receiver_ and _External Staker_. They ensure the other side matches the whitelist.
-5. Neither contract accept any more channels. They only accept one channel and only what matches the allow list.
+2. Admin deploys _Converter_ and _Virtual Staking_ on Consumer side, both referencing each other.
```

Comment:
Are we having a 1 to 1 `converter` / `virtual-staking` relationship? In the prototype, `meta-staking` connected to multiple `mesh-consumer` contracts.

We could have one `virtual-staking` instance per chain, which could be given the ability to deploy new converter contracts.

Reply:
The prototype had lots of issues.

If we have 3 converters virtually staking to one mesh-staking contract, how do we track which rewards go where? Especially when there are caps for different chains?

Exposing these as distinct delegators to the `x/staking` module allows moving a lot of the book keeping there.
There is no real reason to need one.

In other sides, the new "Converter + Virtual Staking Contract" pair is equivalent to the old "mesh-consumer"
And the "Virtual Staking Module" would fit the role of the "meta-staking" contract. Because everything that really needs to be coordinated in one place, is logic that belongs as a native module.